tgs/src/main/cpp/tgs/FeatureExtraction/CalculatorGenome.h

```diff
@@ -37,8 +37,8 @@
 
 namespace Tgs
 {
-  class TGS_EXPORT CalculatorGenome : 
-    public TreeGenome, 
+  class TGS_EXPORT CalculatorGenome :
+    public TreeGenome,
```

Comment:
Spaces at the end of lines are removed by default in QtCreator, and by OCD developers!